shared/common-adapters/box.desktop.js

```diff
@@ -73,38 +56,9 @@ class Box2 extends React.Component<Box2Props> {
     )
   }
 }
-const VBoxGap = ({gap}) => <div style={{flexShrink: 0, height: globalMargins[gap]}} />
-const HBoxGap = ({gap}) => <div style={{flexShrink: 0, width: globalMargins[gap]}} />
 
-const styles = {
-  centered: {alignSelf: 'center'},
-  centeredChildren: {
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-  fullHeight: {height: '100%'},
-  fullWidth: {width: '100%'},
-  vbox: {
-    ...globalStyles.flexBoxColumn,
-    alignItems: 'stretch',
-    justifyContent: 'flex-start',
-  },
-  vrbox: {
-    ...globalStyles.flexBoxColumnReverse,
-    alignItems: 'stretch',
-    justifyContent: 'flex-start',
-  },
-  hbox: {
-    ...globalStyles.flexBoxRow,
-    alignItems: 'stretch',
-    justifyContent: 'flex-start',
-  },
-  hrbox: {
-    ...globalStyles.flexBoxRowReverse,
-    alignItems: 'stretch',
-    justifyContent: 'flex-start',
-  },
-}
+const vBoxGap = (key, gap) => <div key={key} className={`box2_gap_vertical_${gap}`} />
```

Comment:
note these are just functions and not components anymore. if we do this we actually don't make these wrapped components w/ lifecycles so we can reduce some overhead here